app/src/main/java/com/ncnf/event/Event.java

```diff
@@ -31,10 +31,10 @@
     }
 
     private final UUID uuid;
-    private String ownerId;
-    private final Visibility visibility;
-
+    private final String path;
 
+    private String ownerId;
+    private Visibility visibility;
```

Comment:
Yeah I thought of that, might be useful to keep an Event private before publishing it.